.vscode/launch.json

```diff
@@ -6,6 +6,15 @@
     }
   ],
   "configurations": [
+  {
+    "name": "Attach by Process ID",
```

Comment:
Can you document this in the README.md? I'd love to know how to use it, and it'll be good for new developers.